tests/ActivityloggerTest.php

```diff
@@ -100,25 +100,6 @@ public function it_can_log_an_activity_with_a_causer()
     }
 
     /** @test */
-    public function it_can_log_an_activity_with_a_causer_when_there_is_no_web_guard()
-    {
-        config(['auth.guards.web' => null]);
```

Comment:
Well, some users actually don't have a such a guard. https://github.com/spatie/laravel-activitylog/pull/207

Please restore this test.